src/coffee/exportmapping.coffee

```diff
@@ -181,9 +181,19 @@ class ExportMapping
       if price.validUntil
         validUntilPart = "~#{price.validUntil}"
 
+      tiersPart = ''
+      if price.tiers
+        tiersPart = "%#{@_mapTiers price.tiers}"
+
       if price.discounted?
         discountedPricePart = "|#{price.discounted.value.centAmount}"
-      acc + "#{countryPart}#{price.value.currencyCode} #{price.value.centAmount}#{discountedPricePart}#{customerGroupPart}#{channelKeyPart}#{validFromPart}#{validUntilPart}"
+      acc + "#{countryPart}#{price.value.currencyCode} #{price.value.centAmount}#{discountedPricePart}#{customerGroupPart}#{channelKeyPart}#{validFromPart}#{validUntilPart}#{tiersPart}"
+    , '')
+
+  _mapTiers: (tiers) ->
+    _.reduce(tiers, (acc, priceTier, index) ->
+      acc += GLOBALS.DELIM_TIERS_MULTI_VALUE unless index is 0
+      acc + "#{priceTier.value.currencyCode} #{priceTier.value.centAmount} @#{priceTier.minimumQuantity}"
```

Comment:
Hmm, are we sure that `priceTier.value` only contains `centAmount` and `currenctyCode`? Can't it be both [Money](https://docs.commercetools.com/http-api-types.html#money) and [HighPrecisionMoney](https://docs.commercetools.com/http-api-types.html#highprecisionmoney)? 🤔

Reply:
The `priceTier` is always the same as the `base` price and by default, the `base` price is in `centPrecision(Money)`.

PriceTier can either be `centPrecision(Money)` or `highPrecision(HighPrecisionMoney)` but not both at the same time.

For `csv` export and import for a price with tiers, only the `centAmount` and `currenctyCode` are needed.  If `highPrecision` is required it can be set in the `productType` attributes of the price.

Reply:
Hmm, ok. So what happens if I try to import or export a `highPrecision` price with `highPrecision` in `priceTier`?

Reply:
Yes, it works once the base price is in `highPrecision`.

Reply:
Hmm, this doesn't make sense to me. How can it work? Did you test it? [HighPrecisionMoney](https://docs.commercetools.com/http-api-types.html#highprecisionmoney) got more fields, e.g. `preciseAmount`

Reply:
It works for only the default price `type` which is `centPrecision`.